cmd/manager/main.go

```diff
@@ -140,10 +143,27 @@ func main() {
 		}
 	}
 
+	stopChannel := signals.SetupSignalHandler()
+
+	go func() {
+		log.Info("Creating Tekton installation resources once cache is sync'd")
+		if !mgr.GetCache().WaitForCacheSync(stopChannel) {
+			log.Error(errors.New("timed out waiting for caches to sync"), "")
+			os.Exit(1)
+		}
+		// create or update all NSTemplateTiers on the cluster at startup
+		log.Info("Creating the Tekton installation resource")
+		tektonInstallationCR, err := tektoninstallation.Asset("toolchain.openshift.dev_v1alpha1_tektoninstallation_cr.yaml")
+		if err = toolchain.CreateFromYAML(mgr.GetScheme(), mgr.GetClient(), tektonInstallationCR); err != nil {
+			log.Error(err, "Failed to create the 'TektonInstallation' custom resource during startup")
+		}
+		log.Info("Created/updated the NSTemplateTier resources")
```

Comment:
copy-paste :-) 
```suggestion
		log.Info("Created TektonInstallation resource")
```

Reply:
btw, you didn't answer my question regarding the updates: https://github.com/codeready-toolchain/toolchain-operator/pull/16#discussion_r350845536
Earlier or later, I believe that we will need the logic in place anyway - to properly handle upgrades of the operator.

Reply:
> I guess we don't care about updates for now

Correct, here I just try to create the `TektonInstallation` resource, but if it already exists, then it's fine. Is this ok for you?

Reply:
it's fine for now if there is a plan to add a support also for the update. We should be able to properly handle the updates/upgrades of the toolchain-operator including any changes in the CRD.